aspnetcore/toc.md

```diff
@@ -229,85 +207,111 @@
 ## [Security considerations](xref:signalr/security)
 ## [MessagePack Hub Protocol](xref:signalr/messagepackhubprotocol)
 ## [Streaming](xref:signalr/streaming)
-## [Differences between SignalR versions](xref:signalr/version-differences)
+## [Compare SignalR versions](xref:signalr/version-differences)
```

Comment:
This title is misleading. It compares ASP.NET 4.x SignalR to ASP.NET Core SignalR. As is, the title leads you to believe it compares releases ONLY within ASP.NET Core SignalR.